students/Kate_Koehn/Lesson08/inventory.py

```diff
@@ -0,0 +1,54 @@
+'''
+To summarize: 1. You will create a program to initially create, and subsequently update, a CSV file that lists
+which furniture is rented to which customer (to replace use of the spreadsheet mentioned above).
+2. You will create additionally functionality that will load individual customers rentals.
+'''
+
+import csv
+from functools import partial
+
+
+def add_furniture(invoice_file="", customer_name="", item_code="", item_description="", item_monthly_price=0.00):
+    with open(invoice_file, 'a+') as csv_file:
+        writer = csv.writer(csv_file)
+        row = customer_name, item_code, item_description, item_monthly_price
+        writer.writerow(row)
+
+
+def test_add_data():
```

Comment:
Be careful naming something that is not actually a test with "test_" If you have run a test runner, it might just find it and run it, which is probably not what you want.